src/point_one/fusion_engine/messages/fault_control.h

```diff
@@ -102,6 +102,12 @@ enum class FaultType : uint8_t {
    * Payload format: `uint8_t` (0=disable, 1=enable)
    */
   QUECTEL_TEST = 6,
+  /**
+   * Modify the navigation integrity status.
```

Comment:
Suggest: "Simulate a specified integrity status failure (intended for factory test purposes only)."